homework-3/src/test/java/ReadWriteFile.java

```diff
@@ -0,0 +1,50 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadWriteFile {
+    public static void main(String[] args) throws IOException {
+        String file = "/Users/nastyapiaskovska/projects/lits-at-031-homeworks/homework-3/src/text";
+        String fileToWrite = "/Users/nastyapiaskovska/projects/lits-at-031-homeworks/homework-3/src/textWrite";
```

Comment:
imagine i'm trying to run it on my PC
there definitely no **nastyapiaskovska** folder on it unless you are my crush

Reply:
prefer to use relative paths that starts from root of the project